assets/app/views/directives/osc-key-values.html

```diff
@@ -23,7 +25,15 @@
                  spellcheck="false"
                  osc-input-validator="value">
         </div>
-        <button class="btn btn-default" ng-click="addEntry()" ng-disabled="form.key.$error.oscKeyValid || form.value.$error.oscValueValid">Add</button>
+        <button
+          class="btn btn-default"
+          ng-click="addEntry()"
+          ng-disabled="form.$invalid || !key || !value">
```

Comment:
Yeah, I think that's good. We previously didn't allow an empty value, so not a change in behavior.